www/components/shelf/shelf.js

```diff
@@ -27,47 +24,22 @@ class Shelf extends LitElement {
 
   async connectedCallback() {
     super.connectedCallback();
-    this.collapseAll();
-    this.expandRoute(window.location.pathname);
-  }
-
-  async setupShelf(page) {
-    let list = [];
-
-    if (page && page !== '' && page !== '/') {
-      // TODO remove require call - #275
-      // this.shelfList = require(`./${page}.json`);
-      switch (page) {
-
-        case 'about':
-          list = await import(/* webpackChunkName: 'about' */ '../components/shelf/about.json').then(({ default: data }) => data);
```

Comment:
does this mean all the _.json_ files for the shelf can be deleted now?

Reply:
Yes.